frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js

```diff
@@ -187,6 +188,12 @@ export async function sendToStore(batch: Array<DataBucket>): Promise<void> {
       getResolutions(Store.getState().dataset),
     );
     const byteArray = new Uint8Array(data.buffer, data.byteOffset, data.byteLength);
+    console.log("sendToStore");
+    const compressed = lz4.compress(byteArray);
+    // eslint-disable-next-line no-await-in-loop
+    const compressedBase64 = await byteArrayToBase64(compressed);
+    console.log("frontend uncompressed length: ", byteArray.length, " bytes");
+    console.log("frontend compressed (", compressed.length, " bytes): ", compressedBase64);
     // eslint-disable-next-line no-await-in-loop
     const base64 = await byteArrayToBase64(byteArray);
     items.push(updateBucket(bucketInfo, base64));
```

Comment:
The only thing I noticed is, that the compressed bucket doesn't get used here. Or did you just compare the encoded base64 strings?

Reply:
exactly, that was just my intermediate testing code to compare the outputs :)